docs/topics/api/addons.rst

```diff
@@ -192,6 +180,76 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
         dictionary  Dictionary
     ==============  ==========================================================
 
+-------------
+Versions List
+-------------
+
+.. _version-list:
+
+This endpoint allows you to list all versions belonging to a specific add-on.
+
+.. http:get:: /api/v3/addons/addon/(int:addon_id|string:addon_slug|string:addon_guid)/versions/
```

Comment:
to be honest I feel like at some point we should embrace restricting and using the `addon_guid` as the one true source of add-on identification if that's possible.